### PR TITLE
Classic Gazebo - Vehicle customisation of parameters

### PR DESCRIPTION
**Pull request by frederictaillandier**
Updating documentation regarding the default jinja override parameters for sitl-gazebo-classic

### CODE REVIEW

**Comment by hamishwillee** [comment]
@frederictaillandier This is in draft. When you want someone to have a review, please either ping me or move out of draft.

**Comment by frederictaillandier** [comment]
@hamishwillee yeah, it s because I wanted someone to review [this](https://github.com/PX4/PX4-SITL_gazebo-classic/pull/968) before I could cleanup and merge this one

**Comment by frederictaillandier** [comment]
@hamishwillee @Jaeyoung-Lim  maybe now you can review it :)

File: en/sim_gazebo_classic/README.md [hunk 1]
```diff
@@ -470,6 +470,34 @@ Simply enter the following terminal command:
...
+
+With the instance id representing the instance of the vehicle in case of multi-sim simulation.
+
+mavlink_tcp_port: port the mavlink interface would connect to as tcp, default is 4560
+mavlink_udp_port: port the mavlink interface would connect to as udp, default is 14560
+gst_udp_host: ip to which the gstream rtp video stream gets sent to, default is "127.0.0.1"
+gst_udp_port: ip to which the gstream rtp video stream gets sent to, default is 5600
+video_uri: Mavlink camera URI for SITL, default is "udp://127.0.0.1:5600"
+serial_baudrate: Baudrate of Serial device for FMU, default is 921600
+cam_component_id: mavlink id of the camera, default 100
```
Comment (hamishwillee) [inline en/sim_gazebo_classic/README.md:497]: Markdown appends non-empty lines so this renders as one big paragraph.
```suggestion
- `mavlink_tcp_port`: port the MAVLink interface would connect to as TCP. Default is 4560.
- `mavlink_udp_port`: port the MAVLink interface would connect to as UDP. Default is 14560.
- `gst_udp_host`: ip to which the gstream rtp video stream gets sent to. Default is "127.0.0.1"
- `gst_udp_port`: ip to which the gstream rtp video stream gets sent to. Default is 5600.
- `video_uri`: MAVLink camera URI for SITL. The default is `udp://127.0.0.1:5600`.
- `serial_baudrate`: Baudrate of serial device for FMU. The default is 921600.
- `cam_component_id`: MAVLink id of the camera. The default is 100.
```

File: en/sim_gazebo_classic/README.md [hunk 2]
```diff
@@ -470,6 +470,34 @@ Simply enter the following terminal command:
 gst-launch-1.0  -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' \
 ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink fps-update-interval=1000 sync=false
 ```
+You can also customize the uri where the video stream is sent to, by [customizing the sitl_gazebo_classic parameters](jinja_override)
```
Comment (hamishwillee) [inline en/sim_gazebo_classic/README.md:473]: The URL target here is `jinga_override`. This doesn't appear to be a document - do you mean `video_uri` in the following section?

File: en/sim_gazebo_classic/README.md [hunk 3]
```diff
@@ -470,6 +470,27 @@ Simply enter the following terminal command:
...
+
+
+### Customising the SITL Gazebo Classic Parameters
+
+SITL parameters can be specified for all vehicle instances or separately for every individual vehicle instance.
+These override the equivalent parameters in the main simulation jinja file.
```
Comment (hamishwillee) [inline en/sim_gazebo_classic/README.md:479]: Originally this text was at the end and said "If any of these parameters is used in a jinja vehicle file, 
the parameters defined in the jinja file will be replaced by parameter in the jinja file. "

That's confusing. Can I assume that there is a "main jinja file" people will know about - so we're overwriting some data  that applies everywhere with some other data?

OR are you actually saying "The "all vehicle jinja file" sets all values. The individual jinja files overwrite these"? I.e there is no master file, you're explaining that the main file is used by default if there are missing individual vehicle files?

Comment (hamishwillee) [reply en/sim_gazebo_classic/README.md:479]: @frederictaillandier Can you please check my review comments/edits.

File: en/sim_gazebo_classic/README.md (hunk 2 above)
Comment (hamishwillee) [reply en/sim_gazebo_classic/README.md:473]: @frederictaillandier ^^^

**Comment by hamishwillee** [comment]
I am closing due to lack of response from author - pity, this looks useful. @frederictaillandier we can reopen when/if you want to reengage on this in future.

**Pull request closed without merging**